### ISSUE TITLE
 The .NET CLI did not correctly restore debugger files.

### ISSUE THREAD

**Issue by asadsahi** [dotnet/cli#3151]
## Steps to reproduce
1) Installed latest version of vscode

![image](https://cloud.githubusercontent.com/assets/3470892/15423149/1bb655c2-1e74-11e6-818a-df50f7d910c1.png)

2) Install latest version of c# extension
![image](https://cloud.githubusercontent.com/assets/3470892/15423286/a0fca90c-1e74-11e6-9a17-9d51d92a6a3b.png)

3) When opening an aspnet core project, vscode tries to install .net core debugger, but failing:
![image](https://cloud.githubusercontent.com/assets/3470892/15423194/488fed6a-1e74-11e6-9321-e3a04703e36c.png)

## Expected  behavior
Debugger should be installed successfully.

## Actual behavior
Debugger not installing on windows 7

## Environment data
`dotnet --info` output:

.NET Command Line Tools (1.0.0-rc2-002374)

Product Information:
 Version:     1.0.0-rc2-002374
 Commit Sha:  f82da07fc4

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.1.7601
 OS Platform: Windows
 RID:         win7-x64

**Comment by piotrpMSFT** [comment]
@DustinCampbell have  you seen this before?

**Comment by asadsahi** [comment]
Thanks @piotrpMSFT The issue was resolved. 

Problem on my machine was that I had older version of .net runtime (Version: 1.0.0-rc2-002374) even I installed the latest version, On physical location I had two versions and cli was picking up the old version. I manually deleted the folder and the issue was resolved.

This is what the situation was when both versions exist:
![image](https://cloud.githubusercontent.com/assets/3470892/15567996/8b5e2d44-2321-11e6-9c8b-52e8ab304c25.png)
